src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx

```diff
@@ -3,9 +3,10 @@ import styled from '@emotion/styled';
 import {RouteComponentProps} from 'react-router/lib/Router';
 
 import Button from 'app/components/button';
+import {ExportQueryType} from 'app/components/dataExport';
 import DateTime from 'app/components/dateTime';
 import AsyncView from 'app/views/asyncView';
-import {PageContent} from 'app/styles/organization';
+import Layout from 'app/views/auth/layout';
```

Comment:
The basis for the new download page is the Auth modal, since it is the only other place in the app with nothing but a call-to-action on the pattern background